src/runtime/virtcontainers/hypervisor.go

```diff
@@ -674,6 +674,13 @@ type HypervisorConfig struct {
 	// Initdata defines the initdata passed into guest when CreateVM
 	Initdata string
 
+	// InitdataDigest represents opaque binary data attached to a TEE and typically used
+	// for Guest attestation. This will be encoded in the format expected by QEMU for each TEE type.
+	InitdataDigest []byte
+
+	// The initdata image on the host side to store the initdata
+	InitdataImage string
```

Comment:
Do you mean `InitdataFile` or `InitdataDevice`?

Reply:
`InitdataDevice`

Reply:
I mean the squashFS image file I added some more context. The name might make sense?